pyk/src/pyk/kore/rpc.py

```diff
@@ -304,29 +346,7 @@ def request(self, method: str, **params: Any) -> dict[str, Any]:
         server_addr = self._transport.description()
         _LOGGER.info(f'Sending request to {server_addr}: {old_id} - {method}')
```

Comment:
The logging call can be moved into `Transport` so that `_description can become private there.

Reply:
Fixed.